.github/hooks/pre-commit

```diff
@@ -5,6 +5,12 @@
 # ln -s ../../.github/hooks/pre-commit .git/hooks/pre-commit
 #
 
+REACT_FILES=$(git diff --cached --name-only administration/ | sed 's| |\\ |g')
+
+if [ ! -z "$REACT_FILES" ]; then
+  cd administration && yarn lint && cd ..
```

Comment:
maybe I'm doing something wrong, but if I change any ts files in administration and run this script, if-condition is false and yarn lint command is not executed